dev/EVAndPATH/Microsoft.Process.Environment/EnvironmentManager.idl

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Process.Environment
+{
+    [default_interface]
+    runtimeclass EnvironmentManager
+    {
+        static EnvironmentManager GetForProcess();
+        static EnvironmentManager GetForUser();
+        static EnvironmentManager GetForMachine();
+
+        IMapView<String, String> GetEnvironmentVariables();
+
+        String GetEnvironmentVariable(String name);
+        void SetEnvironmentVariable(String name, String value);
+
+        // Path manipulation
+        void AppendToPath(String path);
+        void RemoveFromPath(String path);
+
+        // PathExt Manipulation
+        void AppendToPathExt(String pathExt);
+        void RemoveFromPathExt(String pathExt);
```

Comment:
[Opinion, External]
Can we perhaps make this more descriptive and less coupled to the way this is implemented? e.g. `AddExecutableFileExtension()`

Reply:
Following this comment, would it a good idea to change [AppendTo/RemoveFrom]Path to AddToPath and RemoveFromPath?

Reply:
[Opinion, External]
This one is tough as the `Append` prefix indicates _where_ in the variable value this new path will go. Unlike `pathext`, position is important here. But the verb inconsistency we now created is less than ideal. [pulls hair out]

`AddToPath` and `RemoveFromPath` might be okay, but maybe consider adding an overload that accepts a positional parameter in the near future? (e.g. `AddToPath(String path, At location)`)

Feel free to also just revert my original suggestion to realign the method names. I'm an external and can only provide entertainment and commentary.

Reply:
I think it is a great suggestion.  The IDL has been changed.  Thank you for suggesting it.